### ISSUE TITLE
Collapse is not working on IE 10+

### ISSUE THREAD

**Issue by codecrafting-net** [twbs/bootstrap#26374]
Collapse is not working properly on IE 10 and IE 11. The component do collapse  but not undo it showing the following error: `Object.keys: argument is not a object`. This happens at line 48 of `bootstrap.bundle.js` using Bootstrap 4.1.0. Just to facilitate, follows the code:

```js
  function _objectSpread(target) {
    for (var i = 1; i < arguments.length; i++) {
      var source = arguments[i] != null ? arguments[i] : {};
      var ownKeys = Object.keys(source); //ERROR HERE

      if (typeof Object.getOwnPropertySymbols === 'function') {
        ownKeys = ownKeys.concat(Object.getOwnPropertySymbols(source).filter(function (sym) {
          return Object.getOwnPropertyDescriptor(source, sym).enumerable;
        }));
      }

      ownKeys.forEach(function (key) {
        _defineProperty(target, key, source[key]);
      });
    }

    return target;
  }
``` 
I was able to reproduce the error on the [collapse Bootstrap page](https://getbootstrap.com/docs/4.1/components/collapse/).

Tested environments:

- Windows 8
- IE 10
- Bootstrap v4.1.0
- Jquery v3.3.1 (not slim)
- Popper (v1.14.3)

and

- Windows 10
- IE 11
- Bootstrap v4.1.0
- Jquery v3.3.1 (not slim)
- Popper (v1.14.3)

**Comment by lianee** [comment]
known since #26266 
fixed in #26291 
not released yet... A lot of people are holding upgrades because of that bug

**Comment by codecrafting-net** [comment]
Great! Sorry I actually searched for the error but do not found it. Version 4.0 have this bug?

**Comment by troytempleman** [comment]
I'm having the same issue with Bootstrap 4 in IE. Why was this closed?

**Comment by troytempleman** [comment]
I just found out this was fixed in [Bootstrap 4.1.1 ](https://blog.getbootstrap.com/2018/04/30/bootstrap-4-1-1/)and I was using 4.1.0.
